sql/core/src/main/scala/org/apache/spark/sql/execution/stat/StatFunctions.scala

```diff
@@ -95,4 +97,28 @@ private[sql] object StatFunctions {
     val counts = collectStatisticalData(df, cols)
     counts.cov
   }
+
+  /** Generate a table of frequencies for the elements of two columns. */
+  private[sql] def crossTabulate(df: DataFrame, col1: String, col2: String): DataFrame = {
+    val tableName = s"${col1}_$col2"
```

Comment:
In my previous comment, I mean this `tableName` is not document. Users need to know the name of the first column to operate.

minor: It would be good to check pandas' OR R's naming for this column and follow one of them.